doc/dev/mountpoints.md

```diff
@@ -124,71 +130,68 @@ Here we will look at a few more.
+system:/elektra/mountpoints/\/hosts/definition/positions/set/precommit/#0 = (="sync")
+system:/elektra/mountpoints/\/hosts/definition/positions/set/commit = (="resolver")
+system:/elektra/mountpoints/\/hosts/definition/positions/set/rollback (="resolver")
 ```
 
 This example is very similar to the first one, but the plugin `other_backend` doesn't use the `postorage` and `prestorage` "positions".
 Instead, there is a `validation` position that is (presumably) called in the appropriate phase.
 The plugin `other_backend` may also impose its own restrictions on plugins configured for the `validation` position.
-For example, it may define that such plugins must not generate, remove or modify keys and provide a different position for such plugins.
+For example, it may define that such plugins must not generate, remove or modify keys and provide a different position for plugins that do so.
 
 ```
-system:/elektra/mountpoints/\/hosts/plugins/#0/name (="db_backend")
-system:/elektra/mountpoints/\/hosts/plugins/#1/name (="network")
+system:/elektra/mountpoints/\/hosts/plugins/network/name (="network")
 
-system:/elektra/mountpoints/\/hosts/backend (="#0")
+system:/elektra/mountpoints/\/hosts/plugins/backend/name (="db_backend")
 
-system:/elektra/mountpoints/\/hosts/db/driver (="postgres")
-system:/elektra/mountpoints/\/hosts/db/server (="127.0.0.1")
-system:/elektra/mountpoints/\/hosts/db/port (="5432")
-system:/elektra/mountpoints/\/hosts/db/user (="admin")
-system:/elektra/mountpoints/\/hosts/db/password (="supersecret")
+system:/elektra/mountpoints/\/hosts/definition/db/driver (="postgres")
+system:/elektra/mountpoints/\/hosts/definition/db/server (="127.0.0.1")
+system:/elektra/mountpoints/\/hosts/definition/db/port (="5432")
+system:/elektra/mountpoints/\/hosts/definition/db/user (="admin")
+system:/elektra/mountpoints/\/hosts/definition/db/password (="supersecret")
 
-system:/elektra/mountpoints/\/hosts/phases/set/prestorage/#0 (="#1")
+system:/elektra/mountpoints/\/hosts/definition/phases/set/prestorage/#0 (="network")
 ```
 
 This example shows an entirely different type of backend.
 The hypothetical `db_backend` is backed by a database.
 In this case it is configured for a PostgreSQL database running on `127.0.0.1:5432` to which we connect as user `admin`.
 
 We also configured the `network` plugin to run in the `prestorage` phase of the `set` operation.
-Which phases can be used and how the must be configured of course depends on `db_backend`.
+Which phases can be used and how they must be configured of course depends on `db_backend`.
 
-One might also imagine there could be a (probably quite complicated) part of the mountpoint definition that defines how the relational tables of the database are mapped into the KDB.
+There might also be a (potentially quite complicated) part of the mountpoint definition that defines how the relational tables of the database are mapped into the KDB.
```

Comment:
Hypothetical, better don't write anything unless @flo91 confirms.

Reply:
The whole "Further Examples" section is purely hypothetical, but I'll remove it.